src/constants/posts.ts

```diff
@@ -3,6 +3,7 @@
 
 export const PostTypes = {
     CHANNEL_DELETED: 'system_channel_deleted',
+    CHANNEL_UNARCHIVED: 'system_channel_restored',
```

Comment:
Minor nit, but maybe we could be consistent with our naming here? Sometimes we're restoring the channel and other times we're unarchiving it.